docs/pages/machine-id/guides/kubernetes.mdx

```diff
@@ -0,0 +1,244 @@
+---
+title: Machine ID with Kubernetes Access
+description: Accessing Kubernetes clusters via Machine ID
+---
+
+In this guide, we will demonstrate how to use Machine ID to access a Kubernetes
+cluster protected by Teleport from an automated service.
+
+With Machine ID, Teleport issues short-lived certificates, tied to a machine
+identity, that can be rotated, audited, and managed with the same access
+controls that Teleport provides for human users.
+
+## Prerequisites
+
+<ScopedBlock scope={["cloud"]}>
+You will need Teleport Cloud 10.1.0 or later.
+</ScopedBlock>
+<ScopedBlock scope={["oss","enterprise"]}>
+You will need to be running the Teleport Proxy and Auth Services, version 10.1.0
+or later.
+</ScopedBlock>
+
+If you have not already connected your Kubernetes cluster to Teleport, follow
+the [Kubernetes Access Getting Started Guide](../../kubernetes-access/getting-started.mdx).
+
+If you're not already familiar with Machine ID, follow the
+[Getting Started Guide](../getting-started.mdx) to familiarize yourself with
+Machine ID. You'll also need `tctl` access to initially configure the bot.
+
+(!docs/pages/includes/tctl.mdx!)
+
+Lastly, ensure both `tbot` and `kubectl` are installed on your Machine ID
+client system. Refer to our [Installation guide](../../installation.mdx) for
+`tbot`, and the [Kubernetes documentation](https://kubernetes.io/docs/tasks/tools/)
+for `kubectl`.
+
+
+## Step 1/3. Create a Machine ID bot and assign permissions
+
+In this example, you'll create a bot named "example" and assign it a role
+granting access to a Kubernetes cluster named "example".
+
+First, create a file named `role.yaml` with the following content:
+```yaml
+kind: role
+metadata:
+  name: machine-id-kube-role
+version: v5
+spec:
+  allow:
+    kubernetes_labels:
+      '*': '*'
+    kubernetes_groups:
+    - example
+    kubernetes_users:
+    - alice
+```
+
+Be sure to configure the `allow` fields to match your environment:
+ - `kubernetes_labels` must be specified to allow access to a matching
+   Kubernetes cluster in your Teleport environment.
+
+ - One (or both) of `kubernetes_groups` and `kubernetes_users` must be specified
+   to so that the bot user may be mapped to a Kubernetes user and/or group.
+
+   If only `kubernetes_groups` is set, `kubernetes_users` will be set to the
+   bot's name by default.
+
+   Note that the specific user and group names are arbitrary and do not need to
+   be explicitly created on the Kubernetes cluster. The running Teleport agent
+   and matching Kubernetes RBAC rules are all that's needed to trust bot users
+   as configured here.
+
+Once finished, create the role in Teleport:
+```code
+$ tctl create -f role.yaml
+```
+
+<Details
+title="Create a Kubernetes Role and RoleBinding"
+opened={true}
+>
+If you don't have an existing Kubernetes `Role` or `RoleBinding` to use, create
```

Comment:
What does "to use" mean here? "If your bot user doesn't already have a Kubernetes `Role` or `RoleBinding`"?

Reply:
I've reworded the sentence to hopefully clear this up